facebook/OpenBIC
# fby4: WF: Revise GPIO and multi function's config

Description:
	Refer GPIO and multi function's config to "WAILUA FALLS_ASIC_BIC_GPIO table_20231006".

Motivation:
	Parts of GPIO and multi function are incorrect.

Test plan:
	1. Check SCU register about multifunction: Pass
	2. Check GPIO config: Pass

Test log:
	1. Check SCU register about multifunction
		uart:~$ platform gpio multifnctl
		[   REG    ]  hi                                      lo
		[0x7e6e2410]  0000 0000  0000 0000  0000 0000  0000 0000
		[0x7e6e2414]  1111 0000  0000 0000  0000 0000  0000 0000
		[0x7e6e2418]  0000 0011  0000 0011  1111 1111  1111 1111
		```
		[0x7e6e2510]  0000 0000  0000 0000  0000 0000  0000 0000
		[0x7e6e251c]  0000 0000  0000 0000  0001 1000  0000 0000
	2. Check GPIO config
		uart:~$ platform gpio list_all
		[0  ] PWR_ON_RST_ASIC1_N                 : OD  | output(O) | 0(0)
		[1  ] SYS_RST_ASIC1_N_R                  : OD  | output(O) | 0(0)
		[2  ] PERST_ASIC1_N_R                    : OD  | output(O) | 0(0)
		```
		[99 ] FM_PWRBRK_PRIMARY_R_N              : PP  | input (I) | 1(1)
		[105] EN_CLK_100M_ASIC1_OSC              : PP  | output(O) | 0(0)
		[107] EN_CLK_100M_ASIC2_OSC              : PP  | output(O) | 0(0)
